offscreen_effect_player/pixel_buffer.hpp

```diff
@@ -26,10 +26,10 @@ namespace bnb
         virtual void get_texture(oep_texture_cb callback) override;
     private:
         oep_wptr m_oep_ptr;
-        uint8_t lock_count = 0;
+        uint8_t lock_count {0};
 
-        uint32_t m_width = 0;
-        uint32_t m_height = 0;
+        uint32_t m_width {0};
```

Comment:
maybe that was unnecessary, but I'm ok with it, even don't like such style in general (especially for such simple cases).
don't revert it, leave as is